app/controllers/CreateSeriesController.scala

```diff
@@ -37,10 +36,7 @@ class CreateSeriesController @Inject()(
     val seriesData = createSeriesForm.bindFromRequest.get
     val createSeriesInput = CreateSeriesInput(seriesData.seriesName, seriesData.seriesDescription)
 
-    val accessToken = request.attrs.get(TDRAttributes.OAuthAccessTokenKey).get.accessToken
```

Comment:
It looks like `TDRAttributes.OAuthAccessTokenKey` is still being used in a few other places. Should they be deleted too, and then we can get rid of `TDRAttributes` as well?

Reply:
Sorry, I've just pulled the latest master and seen the Cognito/Silhouette changes.